utils/bcm27xx-eeprom/patches/0001-rpi-eeprom-update-OpenWrt-defaults.patch

```diff
@@ -0,0 +1,26 @@
+From f3adb004f777088386cc6a830711b46635a7516c Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?=C3=81lvaro=20Fern=C3=A1ndez=20Rojas?= <noltari@gmail.com>
+Date: Mon, 23 Mar 2020 10:10:55 +0100
+Subject: [PATCH] rpi-eeprom-update: OpenWrt defaults
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Signed-off-by: Álvaro Fernández Rojas <noltari@gmail.com>
+---
+ rpi-eeprom-update-default | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+--- a/rpi-eeprom-update-default
++++ b/rpi-eeprom-update-default
+@@ -1,8 +1,9 @@
+ 
+ FIRMWARE_ROOT=/lib/firmware/raspberrypi/bootloader
+-FIRMWARE_RELEASE_STATUS="critical"
++FIRMWARE_RELEASE_STATUS="stable"
+ FIRMWARE_IMAGE_DIR="${FIRMWARE_ROOT}/${FIRMWARE_RELEASE_STATUS}"
+ FIRMWARE_BACKUP_DIR="/var/lib/raspberrypi/bootloader/backup"
+ BOOTFS=/boot
+ USE_FLASHROM=0
+ EEPROM_CONFIG_HOOK=
++VCMAILBOX=/usr/bin/vcmailbox
```

Comment:
Can you explain this patch?

Reply:
Using critical releases may be a bit too much conservative and in my opinion using stable releases is safe enough.
VCMAILBOX is added in order to properly locate it, since the default location is /opt/vc/bin/vcmailbox:
https://github.com/raspberrypi/rpi-eeprom/blob/master/rpi-eeprom-update#L34
However, in OpenWrt it's installed under /usr/bin:
https://github.com/openwrt/openwrt/blob/master/package/utils/bcm27xx-userland/Makefile#L60

Reply:
If I can use this on a RaspberryPi I'd give it a try and merge it afterwards.

Reply:
BTW, in my latest update I've also removed the beta firmwares directory.
Most distributions omit it and if a user really wants to play with beta firmwares it will probably be better to force him to manually download them.
This also reduces the package size a lot.

Reply:
I suggest patching the [default value](https://github.com/raspberrypi/rpi-eeprom/blob/a9ca308223c1d0426b9ab320696b95954078c3b4/rpi-eeprom-update#L34) in the script instead of adding the variable here.

Reply:
No, I'm not patching the default value in the script, because that's what the default include script is for.